src/main/java/se/iths/springbootgroupproject/controllers/WebController.java

```diff
@@ -17,9 +21,11 @@
 public class WebController {
 
     private final MessageService messageService;
+    private final UserRepository userRepository;
 
-    public WebController(MessageService messageService) {
+    public WebController(MessageService messageService,UserRepository userRepository) {
         this.messageService = messageService;
+        this.userRepository = userRepository;
```

Comment:
Why have you chosen to use UserRepository and not UserService here? Is it because you are only demoing the user profiles?

Reply:
> Why have you chosen to use UserRepository and not UserService here? Is it because you are only demoing the user profiles?

Yes that is correct, we did not want to do too many changes because of the "demo" like adding even more to service and such. The repository is only used to get the saved user from repo to show that it was indeed saved.